src/babel-serializer/geometry/off.hh

```diff
@@ -1,3 +1,43 @@
 #pragma once
 
-// TODO
+#include <cstddef> // std::byte
+
+#include <clean-core/span.hh>
+#include <clean-core/typedefs.hh>
+#include <clean-core/vector.hh>
+
+#include <typed-geometry/tg-lean.hh>
+
+#include <babel-serializer/errors.hh>
+
+namespace babel::off
+{
+struct read_config
+{
+    bool has_colors = false;
+    bool has_normals = false;
+    bool has_tex_coords = false;
```

Comment:
why is this a configuration that is specified before? I would have expected that the parser does its best and e.g. reports if it has normals in the result

Reply:
I expect that if a header is present that it is correct, and the `read_config` can be ignored. Otherwise it might be necessary to provide that information manually.